### ISSUE TITLE
requirements.txt is missing feedparser and html2text entries (needed for hackernews.py)

### ISSUE THREAD

**Issue by megagumbo** [jquast/x84#236]
I've just done some testing of the new 2.0 release and noticed that hackernews.py threw an exception due to missing 'feedparser' and 'html2text' Python modules. After installing these two manually using 'pip' everything works just fine. In order to easily set up the environment, I'd request to have those missing modules added to requirements.txt

**Comment by jquast** [comment]
Why/How are you using requirements.txt? do you just decide for yourself to use it? Why aren't you installing x/84 as a package, either from pip/pypi, or from "setup.py develop -U" ? In which case, these are described in setup.py.

If anything, the requirements.txt file should be deleted.

**Comment by megagumbo** [comment]
Well, I just did a git pull and pulled all recent changes from there. I do not use virtualenv on my development system and just pull all changes from my cloned repo as well as from upstream. maybe - and I am quite sure this is the case - I do misunderstand how to proplerly set up my dev environment. I did follow the "developers" section in the docs though. However, if this is not really a problem due to me not getting things right, feel free to close this issue.

Ah, and thanks for merging the recent lightbar_main.py changes. There's more to come.
